server/internal/handler/eatery.go

```diff
@@ -19,11 +20,29 @@ func (h *Handler) GetEateries(c echo.Context, params schema.GetEateriesParams) e
 }
 
 // PostEateries implements schema.ServerInterface.
-func (h *Handler) PostEateries(c echo.Context, params schema.PostEateriesParams) error {
-	return c.JSON(http.StatusNotImplemented, schema.Error{
-		Code:  "NOT_IMPLEMENTED",
-		Error: "PostEateries endpoint is not implemented yet",
-	})
+func (h *Handler) PostEateries(c echo.Context, params schema.GetEateriesParams) error {
+	var req schema.EateryCreate
+	if err := c.Bind(req); err != nil {
```

Comment:
これは僕が書いたサンプルコードが間違っていたのでしょうがないんですが、`&`を付けないと正しく動きません（なんでか知りたかったら「Golang ポインタ」とかでググってみてください）
```suggestion
	if err := c.Bind(&req); err != nil {
```